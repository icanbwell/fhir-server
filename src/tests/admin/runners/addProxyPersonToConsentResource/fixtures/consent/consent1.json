{
  "resourceType": "Consent",
  "id": "e0363690-3953-4a02-a98b-f341fd13fb8c",
  "meta": {
    "versionId": "1",
    "source": "medstar",
    "lastUpdated": "2022-09-08T14:05:09.000Z",
    "security": [
      {
        "system": "https://www.icanbwell.com/access",
        "code": "medstar"
      },
      {
        "system": "https://www.icanbwell.com/owner",
        "code": "medstar"
      }
    ]
  },
  "status": "active",
  "scope": {
    "coding": [
      {
        "system": "http://terminology.hl7.org/CodeSystem/consentscope",
        "code": "patient-privacy"
      }
    ]
  },
  "category": [
    {
      "coding": [
        {
          "system": "http://loinc.org",
          "code": "59284-0"
        }
      ]
    }
  ],
  "patient": {
    "reference": "Patient/784991a1-14fe-493c-b5e2-027eb36aa192"
  },
  "dateTime": "2022-09-08T14:05:07.350Z",
  "sourceReference": {
    "reference": "QuestionnaireResponse/fdf49831-d3c0-4a89-9459-e96c6c3beb0f"
  },
  "provision": {
    "type": "permit"
  }
}
