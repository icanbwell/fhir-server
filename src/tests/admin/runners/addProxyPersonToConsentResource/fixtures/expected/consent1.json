{
  "resourceType": "Consent",
  "id": "09d2db52-d386-497f-a777-afe458c7c80b",
  "meta": {
    "versionId": "1",
    "lastUpdated": "2023-08-16T08:07:02.000Z",
    "source": "walgreens",
    "security": [
      {
        "system": "https://www.icanbwell.com/access",
        "code": "walgreens"
      },
      {
        "system": "https://www.icanbwell.com/owner",
        "code": "walgreens"
      },
      {
        "system": "https://www.icanbwell.com/sourceAssigningAuthority",
        "code": "walgreens"
      }
    ]
  },
  "identifier": [
    {
      "id": "sourceId",
      "system": "https://www.icanbwell.com/sourceId",
      "value": "09d2db52-d386-497f-a777-afe458c7c80b"
    },
    {
      "id": "uuid",
      "system": "https://www.icanbwell.com/uuid",
      "value": "09d2db52-d386-497f-a777-afe458c7c80b"
    }
  ],
  "status": "active",
  "scope": {
    "coding": [
      {
        "system": "http://terminology.hl7.org/CodeSystem/consentscope",
        "code": "patient-privacy"
      }
    ]
  },
  "category": [
    {
      "coding": [
        {
          "system": "http://loinc.org",
          "code": "59284-0"
        }
      ]
    }
  ],
  "patient": {
    "extension": [
      {
        "id": "sourceId",
        "url": "https://www.icanbwell.com/sourceId",
        "valueString": "Patient/cde3a031-fd87-4956-a246-7da32dc0aa39"
      },
      {
        "id": "uuid",
        "url": "https://www.icanbwell.com/uuid",
        "valueString": "Patient/cde3a031-fd87-4956-a246-7da32dc0aa39"
      },
      {
        "id": "sourceAssigningAuthority",
        "url": "https://www.icanbwell.com/sourceAssigningAuthority",
        "valueString": "walgreens"
      }
    ],
    "reference": "Patient/cde3a031-fd87-4956-a246-7da32dc0aa39"
  },
  "dateTime": "2022-09-08T14:05:07.350Z",
  "sourceReference": {
    "extension": [
      {
        "id": "sourceId",
        "url": "https://www.icanbwell.com/sourceId",
        "valueString": "QuestionnaireResponse/fdf49831-d3c0-4a89-9459-e96c6c3beb0f"
      },
      {
        "id": "uuid",
        "url": "https://www.icanbwell.com/uuid",
        "valueString": "QuestionnaireResponse/fdf49831-d3c0-4a89-9459-e96c6c3beb0f"
      },
      {
        "id": "sourceAssigningAuthority",
        "url": "https://www.icanbwell.com/sourceAssigningAuthority",
        "valueString": "walgreens"
      }
    ],
    "reference": "QuestionnaireResponse/fdf49831-d3c0-4a89-9459-e96c6c3beb0f"
  },
  "provision": {
    "type": "permit",
    "actor": [
      {
        "role": {
          "coding": [
            {
              "system": "http://terminology.hl7.org/3.1.0/CodeSystem-v3-RoleCode.html",
              "code": "AUT"
            }
          ]
        },
        "reference": {
          "extension": [
            {
              "id": "sourceId",
              "url": "https://www.icanbwell.com/sourceId",
              "valueString": "Patient/person.97f5479a-1611-4964-911f-df1640f454f0"
            },
            {
              "id": "uuid",
              "url": "https://www.icanbwell.com/uuid",
              "valueString": "Patient/person.97f5479a-1611-4964-911f-df1640f454f0"
            },
            {
              "id": "sourceAssigningAuthority",
              "url": "https://www.icanbwell.com/sourceAssigningAuthority",
              "valueString": "bwell"
            }
          ],
          "reference": "Patient/person.97f5479a-1611-4964-911f-df1640f454f0"
        }
      }
    ]
  }
}
